config/taxonomy/application.values.schema.json

```diff
@@ -0,0 +1,73 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "application.values.schema.json",
+    "title": "Application Values Taxonomy",
+    "description": "Values contributed to by the mesh for data applications.",
+    "definitions": {
+        "intent": {"$ref": "policymanager.structs.schema.json#/definitions/intent"},
+        "role": {"$ref": "policymanager.structs.schema.json#/definitions/role"},
+        "app_info": {
+            "type": "object",
+            "properties": {
+                "intent": { "$ref": "policymanager.structs.schema.json#/definitions/intent"},
```

Comment:
do you need intent in "definitions"? can you reference in line 12 directly policymanager.structs.schema.json#/definitions/intent" ?

Reply:
Since we're not referencing that line anymore, I removed intent and role from the definitions